packages/example/src/1week/URL/addQuery/__submit__/minsoo-web.ts

```diff
@@ -0,0 +1,65 @@
+const hasIncludes = (url: string, value: string): 'YES' | 'NO' =>
+  url.includes(value) ? 'YES' : 'NO'
+
+export const splitHashPart = (
```

Comment:
```suggestion
export const getHashParts = (
```

면 어떨까용? split은 Array.split메소드랑 비슷한 네이밍이라 Array를 리턴하는 함수로 오해 받을 수 있을 거 같아요

Reply:
오 좋네요..! 동의합니다. 반영해볼게요!

Reply:
[review: change prefix split -> get](https://github.com/function-and-mountain/functional-coding-nutshell/pull/16/commits/647b616edd01c3590bd5e8c7ce5b47dcc57d248c) 반영해봤습니다!!

Reply:
nonHash도 같이 return 해주고 있어서, get 대신 parse나 decode 같은것도 좋아보입니다!

Reply:
```ts
const { hash, nonHash } = getHashParts("https://naver.com")
const { hash, nonHash } = parseHashParts("https://naver.com")
const { hash, nonHash } = decodeHashParts("https://naver.com")
```

ㅜㅜ 네이밍만 봐서는 저는 다 좋아보여서,, 챗지피티한테 물어봤는데 이렇게 답해주더라구요

<img width="718" alt="image" src="https://github.com/function-and-mountain/functional-coding-nutshell/assets/57122180/3a0d0dc7-bce4-4d89-8b54-94ea64659ba6">

파싱은 해시만 뜯어내는 역할이 아니라, 다른 작업도 해줘야 할 것 같고,
디코드는 인코딩된 애를 디코딩해주는 역할도 포함인 것 같아서
지금 제 함수 역할은 단순히 # 뒤의 문자열부터 뜯어내는 역할이라, get이 조금 더 명확한 것 같은데 민수님 의견도 궁금합니다!

Reply:
아 그러네요 parse나 decode는 hash가 아니라 url이 대상이라 좀 어색해지는군요!
그렇다면 get이 좋아보입니다ㅎㅎ

제가 걸렸던건 { hash: string; nonHash: string; } 이 객체에 hash가 있는데 그 객체 자체를 hash라고 부르니까 헷갈릴까봐서였는데,
지금 다시 보니까 hashParts라고 해서 nonHash도 포함된 걸로 볼 수 있었겠네요. ☺️

Reply:
맞습니다! 민수님이 짚어주신 부분이 저도 공감되어서 고민해봤는데 get이 지금은 제일 무난해보입니다..! 좋은 의견 감사해요!